jekyll/_cci2/monitoring.adoc

```diff
@@ -54,7 +60,69 @@ https://www.nomadproject.io/docs/telemetry/metrics.html#job-metrics[Nomad job me
 |===
 
 When the Nomad metrics container is running normally, no output will be written to standard output or standard error. Failures will elicit a message to standard error.
+// What is meant by Standard Output and Standard Error?
```

Comment:
typically we mean stdout and stderr but they refer to https://en.wikipedia.org/wiki/Standard_streams . I wouldn't worry about it honestly, most people who are reading about metrics should understand the terminology.